app/controllers/api/tags_controller.rb

```diff
@@ -10,7 +10,7 @@ def create_resource_tags(type, _id, data)
       end
       begin
         entry = category.add_entry(data)
-        raise BadRequestError, "#{entry.errors.full_messages.join(', ')}" unless entry.valid?
+        raise BadRequestError, (entry.errors.full_messages.join(', ')).to_s unless entry.valid?
```

Comment:
why the `to_s` here but not in identical code in `app/controllers/api/subcollections/tags.rb`?